common/djangoapps/pipeline_mako/templates/static_content.html

```diff
@@ -78,27 +78,38 @@
 %>${source | n, decode.utf8}</%def>
 
 <%def name="require_module(module_name, class_name)">
```

Comment:
Can you add a docstring/comment for both of these functions describing when to use one over the other.